health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/MyBillsSearch.js

```
@@ -0,0 +1,56 @@
+        title: '',
+    });
+
+    const handleFilterChange = (data) => {
+        setDateRange(data.range);
+    };
+
+
+    return (
+        <Card variant="search" style={{ marginBottom: "1.5rem" }}>
+            <div style={{ width: "80%" }}>
+                <TextBlock body={`${t("HCM_AM_BILL_ID")} *`}></TextBlock>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)